hazelcast/src/main/java/com/hazelcast/sql/impl/exec/CreateExecPlanNodeVisitor.java

```diff
@@ -300,14 +362,24 @@ public void onMapIndexScanNode(MapIndexScanPlanNode node) {
                     node.getIndexName(),
                     node.getIndexComponentCount(),
                     node.getIndexFilter(),
-                    node.getConverterTypes()
+                    node.getConverterTypes(),
+                    node.getDescending()
                 );
             }
         }
 
         push(res);
     }
 
+    @Override
+    public void onUnicastSendNode(UnicastSendPlanNode node) {
+        Outbox[] outboxes = prepareOutboxes(node);
```

Comment:
As mentioned before, the partitioner is simply ignored here. We need unit tests to ensure that the visitor produces the expected node. See `CreateExecPlanNodeVisitorTest`.

Reply:
Will be part of the generalized node change.